### PR TITLE
Course selection styles1

### PR DESCRIPTION
**Pull request by jerrydev91**
Fixed some styling.. more work still pending on this. Take a look at this advance ;)

### CODE REVIEW

File: components/FlatListHorizontalDetail2.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  imageUrl,
+  percentOff,
+  courseInitials,
+  courseName,
+  courseDistance
+  }) => {
+  return (
+    <View>
+      <Image style={styles.imageAsContainer} source={{ uri: `${imageUrl}`}}>
+        <TouchableOpacity style={styles.button} onPress={console.log("pressing")}>
```
Comment (peterpme) [inline components/FlatListHorizontalDetail2.js]:
```

const FlatList = ({
 onPress,
}) => (
<TouchableOpacity onPress={onPress} />
)

FlatList.defaultProps = {
onPress: () => { console.log('pressing') }
}

```

that way, whenever you pass down the correct one, you dont' have to go back and change it. it'll be done!

File: components/FlatListHorizontalDetail2.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+          courseName
+        </Text>
+        <Text style={{ marginTop: 5, fontSize: 10, color: 'gray' }}>
+          • numHoles Holes • PAR numPAR • Free Carts
+        </Text>
+        <View style={{ flexDirection: 'row', justifyContent: 'flex-start', marginTop: 10 }}>
+          <Text style={{ fontSize: 14, color: 'green' }}>
+            $200
+          </Text>
+          <Text style={{ fontSize: 10, color: 'gray', marginLeft: 15 }}>
```
Comment (peterpme) [inline components/FlatListHorizontalDetail2.js]: make sure you remove styles from here and place them in `StyleSheet`

File: components/NearbyFlatListDetail.js [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+            </View>
+          </View>
+          <View style={styles.bottomViewContainer}>
+            <View style={{ flexDirection: 'row', justifyContent: 'center', alignItems: 'center' }}>
+              <View style={styles.initialsView}>
+                <Text style={styles.initials}> {courseInitials} </Text>
+              </View>
+              <View style={styles.courseDescription}>
+                <Text style={styles.courseName}> {courseName} </Text>
+                <Text style={styles.distance}> {courseDistance} minutes/hours away </Text>
```
Comment (peterpme) [inline components/NearbyFlatListDetail.js]: you can get rid of `minutes/hours away` here and just have `{courseDistance} mi` (for miles)

File: components/SectionContent.js [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  return (
+    <View style={styles.container}>
+      <View style={styles.textView}>
+        <Text style={styles.textTitle}>
+          {title}
+        </Text>
+      </View>
+      <View style={styles.btnView}>
+        <TouchableOpacity style={styles.btn}
+          onPress={() => this._onPressSectionBtn()}>
```
Comment (peterpme) [inline components/SectionContent.js]: this doesn't work :)

**Comment by jerrydev91** [comment]
Hey! Nice, thanks for pointing out those details. I'm gonna fix them too in the next PR. Im just going to work on the Navigation as you suggested there in the Slack app :)

**Pull request merged**